addons/mail/static/src/components/link_preview_aside/link_preview_aside.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.LinkPreviewAsideView" owl="1">
+        <button t-if="linkPreviewAsideView.displayClose and linkPreviewAsideView.linkPreview.isDeletable" class="o_LinkPreviewAside btn" t-on-click="linkPreviewAsideView.onClickUnlink" t-attf-class="{{ className }}" t-ref="root">
```

Comment:
template does nothing if this condition is false, should consider moving condition to parents instead

`linkPreviewAsideView` should be computed depending on whether it needs to be displayed or not, and condition should be on its existence in parent

Reply:
t-on after t-aff-class